src/assets/translations/en/main.json

```diff
@@ -2983,6 +2983,18 @@
       "claimAvailable": "Claim Available",
       "unknown": "Unknown"
     },
+    "swap": {
+      "processing": "Your swap of %{sellAmountAndSymbol} to %{buyAmountAndSymbol} is still being processed.",
```

Comment:
`sellAmountAndSymbol` to parts across the board 🙏🏽

Reply:
Actually after reading the PR I think that's absolutely fine

Reply:
Yes, leveraging JSX to format is is quite convenient even thought not so descriptive 🙏